enrich_test.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"github.com/jfrog/jfrog-cli-security/commands/enrich/enrichgraph"
+	securityTests "github.com/jfrog/jfrog-cli-security/tests"
+	securityTestUtils "github.com/jfrog/jfrog-cli-security/tests/utils"
+	"github.com/stretchr/testify/assert"
+	"path/filepath"
+	"testing"
+)
+
+func testVulns(t *testing.T, vulns []struct {
+	BomRef string
+	Id     string
+}) {
+	for _, vuln := range vulns {
+		assert.NotEqual(t, vuln.BomRef, nil)
+		assert.NotEqual(t, vuln.Id, nil)
+	}
+}
```

Comment:
```suggestion
func testVulns(t *testing.T, vulns Vulnerabilities) {
	for _, vuln := range vulns {
		assert.NotEqual(t, vuln.BomRef, nil)
		assert.NotEqual(t, vuln.Id, nil)
	}
}
```
Use the structs from the formats that you added. you dont need anonymus structs

Reply:
I want the testVulns function to receive both xml and json structs. I only care about specific fields in this so I used annonymous structs to make this function generic. whats wrong with that ?